src/index.ts

```diff
@@ -48,6 +48,7 @@ async function resolve<T, U>(root: T, parts: string[]): Promise<U> {
 export async function generateOpenAPITypes(
   schema: OpenAPIV3.Document,
   baseName = 'API',
+  filterStatuses: number[] = [],
```

Comment:
I would rather create an `options` object with a `filterStatuses` property in order to be able to create many options without adding a new parameter.